opentargets.json

```
@@ -1122,13 +1119,63 @@
         "iPSC derived cortical neurons"
       ]
     },
-    "biomarker": {
+    "biomarkerName": {
       "type": "string",
-      "description": "Altered characteristic that influences the disease process.",
+      "description": "Altered characteristics that influences the disease process.",
       "examples": [
-        "ABL1:E275K"
+        "BRAF (V600E,V600D,V600K,V600M,V600G,V600R)"
       ]
     },
+    "biomarkers": {
+      "type": "object",
+      "description": "List of biomarkers.",
+      "properties": {
+        "variant": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "examples": [
+                  "BRAF:V600M"
+                ]
+              },
+              "id": {
+                "$ref": "#/definitions/variantId"
+              },
+              "variantFunctionalConsequenceId": {
+                "$ref": "#/definitions/variantFunctionalConsequenceId"
+              }
+            }
+          },
+          "uniqueItems": true
+        },
+        "geneExpression": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
```

Adding a comment for the `geneExpression.name` field:

```suggestion
                "type": "string",
                "description": "Raw gene expression annotation from the source",
```